src/Http/Middleware/ShareInertiaData.php

```diff
@@ -42,7 +42,9 @@ public function handle($request, $next)
                 }
 
                 if (Jetstream::hasTeamFeatures() && $request->user()) {
-                    $request->user()->currentTeam;
+                    $request->user()->setRelation('currentTeam', $request->user()->allTeams()->filter(function ($team) use ($request) {
+                        return $team->id === $request->user()->current_team_id;
+                    })->first());
```

Comment:
Hi,

I think that PR cause a bug due to current_team_id field who can be null. Just try to install a fresh Laravel with Jetstream and you're unable to login (white page due to JS error and team).

It's because field is refresh with switchTeam method. Before PR, currentTeam() was called and check that.

Maybe you can try that 👍

```
if (is_null($request->user()->current_team_id)) {
    $request->user()->switchTeam($request->user()->personalTeam());
}

return $team->id === $request->user()->current_team_id;
```

Reply:
Thanks, reverted.